packages/site-admin/src/router/components/link/index.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { useLink } from '../../hooks';
+import { type NavigationOptions } from '../../types';
+
+type LinkProps = React.ComponentPropsWithoutRef< 'a' > & {
+	to: string;
+	options?: NavigationOptions;
+	children: React.ReactNode;
+};
+
+export function Link( { to, options, children, ...props }: LinkProps ) {
+	const { href, onClick } = useLink( to, options );
+
+	return (
+		<a href={ href } onClick={ onClick } { ...props }>
+			{ children }
+		</a>
+	);
+}
```

Comment:
The idea is to expose the `Link` component for use right? In this case, you'll need to export it in `packages/site-admin/src/index.ts`. And if thats the case, also a description in the README would be good.

Reply:
That's correct. Let's expose it.

Reply:
https://github.com/Automattic/wp-calypso/pull/101820/commits/2d71c238986b70fc3e4c89ee273bf647e26c3340

Added also in the Changelog file. I'm going to create an issue to document the API of the package, too.